README.md

```diff
@@ -1,14 +1,28 @@
-# Laravel Scout MeiliSearch
-
-[![Licence](https://img.shields.io/badge/licence-MIT-blue.svg)](https://img.shields.io/badge/licence-MIT-blue.svg)
-[![Actions Status](https://github.com/meilisearch/meilisearch-php/workflows/Tests/badge.svg)](https://github.com/meilisearch/meilisearch-php/actions)
-[![Latest Stable Version](https://poser.pugx.org/meilisearch/meilisearch-laravel-scout/version)](https://packagist.org/packages/meilisearch/meilisearch-laravel-scout)
-
-The Laravel scout package for MeiliSearch.
-
-MeiliSearch provides an ultra relevant and instant full-text search. Our solution is open-source and you can check out [our repository here](https://github.com/meilisearch/MeiliSearch).
-
-Here is the [MeiliSearch documentation](https://docs.meilisearch.com/) 📖
+<p align="center">
+  <img src="https://res.cloudinary.com/meilisearch/image/upload/v1587402338/SDKs/meilisearch_laravel.svg" alt="MeiliSearch-Laravel Scout" width="200" height="200" />
```

Comment:
Is that logo the real logo of [Laravel Scout](https://github.com/laravel/scout)?

Reply:
Sorry, I don't know that Laravel Scout has a different logo than Laravel :/

Reply:
Np 🙂

Reply:
Done 👍